dso-l1/src/main/java/com/tc/object/TCObjectServerMapImpl.java

```diff
@@ -1169,18 +1169,19 @@ public void addTxnInProgressKeys(Set addSet, Set removeSet) {
 
   @Override
   public void doRegisterListener(Set<ServerEventType> eventTypes, boolean skipRejoinChecks) {
-    Set<Object> params = new HashSet<Object>();
+    Set<Object> params = new HashSet<>();
```

Comment:
This array should be in order. The old code is only working because it's order agnostic.